sdk/js/src/Client.ts

```diff
@@ -132,6 +138,11 @@ export class DVCClient implements Client {
                     )
                 ).then((config) => {
                     this.config = config as BucketedUserConfig
+
+                    if (this.options?.enableCloudData === true || this.config?.project.settings?.cloudEntityData?.enabled) {
+                        saveEntity(this.user, this.environmentKey).then((res) => console.log(`Saved response entity! ${res}`))
```

Comment:
can we fire this off in parallel to the config fetch rather than doing one and then the other? It would be faster that way